docs/book/src/tasks/experimental-features/runtime-sdk/implement-extensions.md

```diff
@@ -56,7 +56,7 @@ import (
 )
 
 var (
-	catalog = runtimecatalog.New()
+	catalog  = runtimecatalog.New()
```

Comment:
nit non blocking
```suggestion
	catalog = runtimecatalog.New()
```

Reply:
Oh thx. Not sure how that happened

Reply:
I would keep it. It's the correct formatting in go that Intellij automatically applied. Please note the next line
```go
	catalog  = runtimecatalog.New()
	setupLog = ctrl.Log.WithName("setup")
```